docs/develop/ts/service-communication.mdx

```diff
@@ -51,6 +59,11 @@ CODE_LOAD::ts/src/develop/service_communication.ts#<start_request_response_objec
     In case of failures, Restate takes care of retries.
 </Admonition>
 
+<Admonition type="info" title="Workflow retention time">
+    Once the `run` handler has finished, the other handlers can still be called up to the retention time of the workflow, by default 24 hours.
+    This can be configured via the [Admin API](/references/admin-api/#tag/service/operation/get_service) per Workflow definition by setting `workflow_completion_retention`.
+</Admonition>
```

Comment:
This totally looks in the wrong place IMO. I think it would fit best either the _Overview_ page, or the _State_